src/main/scala/pl/multiplex/models/Consumer.scala

```diff
@@ -0,0 +1,23 @@
+package pl.multiplex.models
+
+import com.fasterxml.jackson.annotation.JsonBackReference
+import javax.persistence._
+
+import scala.beans.BeanProperty
+
+@Entity
+class Consumer extends Serializable {
```

Comment:
po co nam serializable?

Reply:
Jest to jedna z wytycznych przy tworzeniu Java Beans. Ja to rozumiem tak, że Serializable pozwala na przetworzenie obiektu na inna postać i ponowne jego odtworzenie.  